rspace-bench/src/main/scala/coop/rchain/rspace/bench/EvalBenchStateBase.scala

```diff
@@ -0,0 +1,59 @@
+package coop.rchain.rspace.bench
+
+import java.io.{FileNotFoundException, InputStreamReader}
+import java.nio.file.{Files, Path}
+import org.openjdk.jmh.annotations.{Setup, TearDown}
+
+import coop.rchain.crypto.hash.Blake2b512Random
+import coop.rchain.models.Par
+import coop.rchain.rholang.interpreter.{Interpreter, Runtime}
+import coop.rchain.shared.PathOps.RichPath
+
+trait EvalBenchStateBase {
+	private val dbDir: Path   = Files.createTempDirectory("rchain-storage-test-")
```

Comment:
It looks like indentation was accidentally increased (or perhaps some tabs snuck in?).